packages/terra-navigation-side-menu/src/MenuItem.module.scss

```diff
@@ -117,17 +117,7 @@
 
         /* stylelint-disable-next-line max-nesting-depth */
         .chevron {
-          color: var(--terra-navigation-side-menu-item-selected-hover-chevron-color, #909496);
-        }
-      }
-
-      &.is-focused {
-        background-color: var(--terra-navigation-side-menu-item-selected-focus-background-color, #f4f4f4);
-        color: var(--terra-navigation-side-menu-item-selected-focus-color);
-
-        /* stylelint-disable-next-line max-nesting-depth */
-        .chevron {
-          color: var(--terra-navigation-side-menu-item-selected-focus-chevron-color, #909496);
+          color: var(--terra-navigation-side-menu-item-selected-hover-chevron-color, #1C1F21);
```

Comment:
Was this color provided by the design team?

Reply:
color contrast change reverted. Thanks
f257333e9538e426d69ce329087fe07892bc97da